glambie/plot/processing_plots.py

```diff
@@ -33,7 +40,7 @@ def plot_all_plots_for_region_data_group_processing(output_path_handler: OutputP
                                       output_filepath=plot_fp,
                                       plot_errors=False)
     # Convert to same unit as annual datasets now
-    data_catalogue_annual_raw = convert_datasets_to_monthly_grid(data_catalogue_annual_raw)
+    # data_catalogue_annual_raw = convert_datasets_to_monthly_grid(data_catalogue_annual_raw)
```

Comment:
Removed by mistake? If so, undo. If not, remove

Reply:
oops, i fully removed it now

Reply:
I don't see the commit for this (last commit 2 days back). Looks like the changes weren't pushed